server/app/utils/auth_helpers.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+def get_redirection_url_for_user(user_role: str):
+
+    if user_role == "student":
+        redirect_url = "http://localhost:%s/StudentHome"%os.getenv("REACT")
```

Comment:
We probably need to test this on the VM because it might redirect to local host. if it does though it's not a big deal we can get the current host and format it